### PR TITLE
ls, touch: add examples

### PR DESCRIPTION
**Pull request by blalyasar**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ls.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Long format list of all files, sorted by modification date (oldest first):
 
 `ls -ltr`
+
+- List Only Directories:
+
+`ls -d */`
```
Comment (bl-ue) [inline pages/common/ls.md:36]:
```suggestion
`ls -d {{*/}}`
```

**Comment by blalyasar** [comment]
I am accepted  CLA. Take it easy . Thanks for attention. for review command

**Comment by blalyasar** [comment]
I am accepted CLA. and accept commit suggestion. Take it easy . Thanks for attention. for review command

File: pages/common/touch.md [hunk 2]
```diff
@@ -14,3 +14,7 @@
 - Use the times from a file to set the times on a second file:
 
 `touch -r {{filename}} {{filename2}}`
+
+- Create multiple files:
+
+`touch app{1,2,3}.txt`
```
Comment (bl-ue) [inline pages/common/touch.md:20]: I'm not sure about this example.  While definitely useful, it seems to demonstrate typical shell functionality (i.e. glob patterns) rather than behavior specific to the `touch` utility itself.  I'll let others way in here, ping @sbrl.

**Comment by bl-ue** [comment]
Also @blalyasar let me tell you:  you're welcome to contribute translations (e.g. Turkish?) to any pages you'd like!

File: pages/common/touch.md (hunk 2 above)
Comment (alexmyczko) [reply pages/common/touch.md:20]: I like the example, find my self using -d for ls often, just make it ls -d */ then.

Comment (bl-ue) [reply pages/common/touch.md:20]: Yeah, I think the `ls -d */` is a great one.  But I'm not so sure about this `touch app{1,2,3}.txt` one.  Let's wait and see what others think.

**Comment by blalyasar** [comment]
@bl-ue  
Turkish is latin alphabet.
Tldr page is not contain complex english .
I dont think translate this pages for turkish.
I am suggestion use tldr,  my friend and other turkish users , 
This link available, https://blalyasar.blogspot.com/2021/01/cok-kullanlmayan-linux-komutlar.html?q=tldr

File: pages/common/touch.md (hunk 2 above)
Comment (einverne) [reply pages/common/touch.md:20]: @bl-ue Since the touch page does not reach the limitation, how about adding this one. I think this example is useful sometimes.

**Review by einverne — APPROVED** [review]
Thank you for the new example, it looks good to me.

File: pages/common/touch.md (hunk 2 above)
Comment (sbrl) [reply pages/common/touch.md:20]: Looks like shell-specific to me. Generally we try to document the features of a command, rather than the shell it's running in.

File: pages/common/ls.md [hunk 3]
```diff
@@ -30,3 +30,7 @@
 - Long format list of all files, sorted by modification date (oldest first):
 
 `ls -ltr`
+
+- List Only Directories:
```
Comment (Waples) [inline pages/common/ls.md:34]:
```suggestion
- Only list directories:
```

**Review by Waples — CHANGES_REQUESTED** [review]

**Comment by marchersimon** [comment]
@blalyasar are you still working on this?

**Comment by blalyasar** [comment]
No I am not working on this. But I am waiting for merge pull request . And Why not accepted pull request . I dont understand.. If you say close pull request. maybe ı am closed to this branch fork.

**Review by sbrl — APPROVED** [review]
Thanks, @blalyasar!

**Review by sbrl — APPROVED** [review]
Thanks, @blalyasar!

**Comment by sbrl** [comment]
Merging, as @Waples' comment appears to be addressed and it's like October in May
